python/ray/tune/tune.py

```diff
@@ -564,6 +567,7 @@ def sigint_handler(sig, frame):
         default_mode=mode)
 
 
+@PublicAPI
 def run_experiments(
```

Comment:
How about all the args, are they all Public or are some more beta/experimental/deprecated?

Reply:
I think we can leave that in the actual docstring though